docs/ecosystem/initiatives.md

```diff
@@ -9,3 +9,6 @@ The Astar SpaceLabs is a separate arm within the BizDev department. The Astar Sp
 
 [Astar Degens](https://astardegens.com/):
 ASTAR DEGENS DAO - Incubating promising projects and bringing new ideas to the Astar Ecosystem with the help of a community-governed DAO fund. Astar Degens is a community without hierarchy. Where impactful action is incentivized and rewarded. We welcome all ideas equally, and value productive effort. As a community, we help realize the value of supportive cooperation within the blockchain space, by encouraging fearless participation within the Astar Network.
+
+Astar Troopers:
```

Comment:
@SUGURUSEO Both Space Labs and Degens have a link to follow for more details, shoud we perhaps link Troopers likewise to some place with more info (maybe Astar Troopers Discord ?) or give any additional instructions how someone can participate?

Reply:
@andabakus I agree! I added a link to the Astar Troopers Discord so that anyone interested in this can join the group.
Thanks for the comment!